frontend/pubspec.yaml

```diff
@@ -61,9 +61,12 @@ dependencies:
 
 dev_dependencies:
   build_runner: ^2.3.3
+  protoc_builder:
+    git:
+      url: https://github.com/fischerscode/dart-protoc-builder.git
```

Comment:
:question: This repo has 0 stars with the last activity 9 months ago, but I guess you picked this on purpose? Could you quickly explain why you chose this over the original repo (which seems to be exactly the same)?

Reply:
Yeah, the `protoc_builder` is in general not a too active repo. I decided to take the state of the PR https://github.com/pikaju/dart-protoc-builder/pull/21 as it seems much more reasonable to specify the protoc_plugin in the pubspec.yaml instead of making protoc_builder download it.